src/features/generalpairfeature.jl

```diff
@@ -0,0 +1,19 @@
+#=
+Feature of a pair of atoms. Currently only used in WeaveModel, but may eventually
+have a version of AtomGraph that allows edge features...
+=#
+
+# GeneralPairFeature describes the general features between any given pair of atoms.
+struct GeneralPairFeature <: AbstractFeature
```

Comment:
I think this can just be renamed `PairFeature` and be a subtype of `AbstractPairFeature`, right?

Reply:
yes. I don't really think there'd be any ambiguity issues there

Reply:
Fixed in 7e53044.